common/tracking/schemas.go

```diff
@@ -0,0 +1,47 @@
+package tracking
+
+import (
+	"time"
+
+	"github.com/bentoml/yatai-schemas/modelschemas"
+)
+
+type CommonProperties struct {
+	OrganisationUID string
+	Timestamp       time.Time
+	YataiVersion    string
+}
+
+type TriggerEvent struct {
+	UserUID string
+}
+
+type LifecycleEvent struct {
+	LifecycleEventType string
```

Comment:
It is recommended that each schema structure field should have `json` tags

Reply:
thank you for the tip :) added it